lib/utils/runOpen.js

```diff
@@ -24,7 +24,7 @@ function runOpen(uri, options, log) {
 
       return open(pageUrl, openOptions).catch(() => {
         log.warn(
-          `${openMessage}. If you are running in a headless environment, please do not use the --open flag`
+          `Unable to open ${pageUrl} in browser${openOptionValue}. If you are running in a headless environment, please do not use the --open flag`
```

Comment:
`: ${options.open}` -> `: "${options.open}"`

`Unable to open ${pageUrl} in browser${openOptionValue}` -> `Unable to open "${pageUrl}" in browser${openOptionValue}`

Reply:
Updated! Thanks.